ipaserver/install/ipa_kra_install.py

```diff
@@ -83,33 +83,13 @@ def validate_options(self, needs_root=True):
     @classmethod
     def get_command_class(cls, options, args):
         if options.uninstall:
-            return KRAUninstaller
+            raise RuntimeError(
+                'Standalone KRA uninstalling was removed in FreeIPA 4.5 as it '
+                'had never worker properly and only caused issues.')
```

Comment:
Perhaps `uninstallation` is the better word here, thanks for finding the typo as well. However, `had` is used properly, check your grammar ;)